README.md

```diff
@@ -131,6 +131,9 @@ The plugin supports the following configuration options in the `format` section
             * With this option in `true`, the formatter will instruct `ktn_dodger` to actually parse the macros.
             * The default value is `true`.
             * The idea is for users to turn it to `false` only for the module that contain macros that would be broken otherwise.
+        + `sort_exported_funcs` (`alphabetically` | `false`)
+            * Specifies if the export list should have be sorted alphabetically (note, that if two functions share the name, they will be sorted by arity).
```

Comment:
```suggestion
            * Specifies if the export list should be sorted alphabetically (note that if two functions share the name, they will be sorted by arity).
```

Reply:
oopsie, that's what I get for trying out different tenses and forgetting to clean up. Thanks!